gui/services/forms.py

```diff
@@ -724,13 +724,10 @@ def __init__(self, *args, **kwargs):
 
     def clean_iscsi_lunid(self):
         lunid = self.cleaned_data.get('iscsi_lunid')
-        if not lunid:
+        if lunid is None:
```

Comment:
This arises in the situation when user removes the defaul 0 present in the lunid field. We can either make the field a compulsion or default it to 0 if it is None. Because when it is None, the API will show it as AUTO for the webclient view

Reply:
What does it do in the backend though? We may need to migrate the values if there is null in the database.